### PR TITLE
Week 5 goals

### LINKED ISSUE #23: [Week 5 Goals]
* Add enemies
* Add rooms
* Refactor Character and Fight class for more efficient code writing

### PR DESCRIPTION
**Pull request by TMGWill**
* Add enemies
* Add rooms
* Refactor Character and Fight class for more efficient code writing

### CODE REVIEW

**Comment by TMGWill** [comment]
Both commits are for my Week 5 Goals.

File: 486_Project/Characters_Class.py [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
     """
     warrior class for player
     """
     def __init__(self):
         # [strength, dexterity, intelligence, constitution]
-        self.base_stats = [7, 5, 5, 7]
+        self.base_stats = [6, 4, 4, 6]
 
         # weapon
         self.weapon = [1, 0, 0, 1, "Ole Trusty Sword"]
```
Comment (walkerje) [inline 486_Project/Characters_Class.py:12]: Does the [1,0,0,1] relate to [str, dex, int, con]? I think this is a great way to implement item buffs, if so!

File: 486_Project/Characters_Class.py [hunk 2]
```diff
@@ -88,6 +91,70 @@ def __init__(self):
         self.enemy_double_cd = 0
 
 
+class Corrupted_knight:
```
Comment (walkerje) [inline 486_Project/Characters_Class.py:94]: Great to see enemies are getting more attributes!

File: 486_Project/Characters_Class.py [hunk 3]
```diff
@@ -97,23 +164,252 @@ def __init__(self):
...
+        self.enemy_max_hp = self.stats[3] * 4
+        self.enemy_hp = self.enemy_max_hp
 
         # enemy menu for choices during combat
-        self.wolf_menu = ["attack", "defend", "skills"]
+        self.enemy_menu = ["attack", "defend", "skills"]
 
         # enemy skills to use during combat
-        self.wolf_skills = ["maul", "rend"]
+        self.enemy_skills = ["maul", "rend"]
```
Comment (walkerje) [inline 486_Project/Characters_Class.py:180]: Non-attacking skills might be an interesting addition here! For example, one thing wolves might do would be "lick wounds", which might heal them slightly.

File: 486_Project/Characters_Class.py [hunk 4]
```diff
@@ -97,23 +164,252 @@ def __init__(self):
...
+
+        # health points
+        self.enemy_max_hp = self.stats[3] * 4
+        self.enemy_hp = self.enemy_max_hp
+
+        # enemy menu for choices during combat
+        self.enemy_menu = ["attack", "defend", "skills"]
+
+        # enemy skills to use during combat
+        self.enemy_skills = ["lil hit", "mace knee"]
```
Comment (walkerje) [inline 486_Project/Characters_Class.py:343]: Loving the skill titles here.

File: 486_Project/Text_Game.py [hunk 5]
```diff
@@ -57,22 +64,118 @@ def select_class():
...
+""")
+Death_Dog_Encounter = Room("""
+A journey of a thousand miles begins with a single step.
+""")
+Dragon_Intro = Room("""
+A journey of a thousand miles begins with a single step.
+""")
+Dragon_Encounter = Room("""
+A journey of a thousand miles begins with a single step.
+""")
```
Comment (walkerje) [inline 486_Project/Text_Game.py:134]: Great that these have proper titles now! They are far easier to distinguish between because of this.

**Review by walkerje — APPROVED** [review]
Great progress. I have left comments throughout the documents for this PR; feel free to accept and merge!

**Pull request merged**